corehq/apps/reports/static/reports/js/edit_scheduled_report.js

```diff
@@ -73,10 +74,14 @@ hqDefine("reports/js/edit_scheduled_report", [
         var showUcrElements = _.any(
             selectedConfigs, function (i) {return isConfigurableMap[i] === true;}
         );
+        var supportTranslation = _.any(
+            selectedConfigs, function (i) {return isSupportTranslations[i] === true;}
+        );
 
-        if (showUcrElements) {
-            $("#ucr-privacy-warning").show();
-
+        if (showUcrElements || supportTranslation) {
```

Comment:
Might be a little clearer if you change the last few lines to something like

```
var showUcrElements = _.any(
    selectedConfigs, function (i) {return isConfigurableMap[i] === true;}
);
var showTranslations = showUcrElements || _.any(
    selectedConfigs, function (i) {return isSupportTranslations[i] === true;}
);

if (showTranslations) {
...
```
That also has the nice side affect of avoiding having multiple variables with similar names after renaming `isSupportTranslations` to `supportsTranslations` as I suggested above.